woocommerce/woocommerce
# Remove unfinished and unused SplitDropdown component

### All Submissions:






### Changes proposed in this Pull Request:

This PR removes the unfinished and unused `SplitDropdown` component. We started work on this component but are not using it, and have a better implementation based on `DropdownMenu` from `@wordpress/components` that is fully functional and styled as we need. As such, `SplitDropdown` is not needed, and removing it from our codebase reduces tech debt and potential confusion.

https://github.com/woocommerce/woocommerce/blob/b9f1f57e43a3ca47e3a0a2807908119b97d28343/plugins/woocommerce-admin/client/products/product-form-actions.tsx#L170

In the future, if we need a similar dropdown button, we should take the implementation mentioned above and pull it out into a generic, reusable component.

Slack discussion: p1663792772173919-slack-C01CMSA1VHS

### How to test the changes in this Pull Request:

1. `pnpm run build` and make sure everything builds
2. `pnpm --filter=@woocommerce/storybook storybook ` and make sure `SplitDropdown` doesn't show up in Storybook

### Other information:




### FOR PR REVIEWER ONLY:
